File: day8/student.java
import java.util.Scanner;
class Student
{
	private int id;
	private String name;
	public Student() // Constructor
	{
		Scanner sc  = new Scanner(System.in);
		System.out.println("Enter the Name : ");
		name = sc.nextLine();
		System.out.println("Enter the id : ");
		id = sc.nextInt();
	}
	void Display() // function to display data
	{
		System.out.println("Name : "+name);
		System.out.println("ID   : "+id);
	}
	public static void main(String[] args) 
	{
		Student s = new Student();
		s.Display();	
	}
}

File: day8/Student.cpp
#include <iostream>
using namespace  std;
class Student
{
	private:
	 int id;
	  string name;
	public:
	Student() // Constructor
	{
		
		name = "Demo";
		id = 13234;
	}
	void Display() // function to display data
	{
		cout<<"\nName is : "<<name;
		cout<<"\nID   is : "<<id<<endl;
	}
};
int main()
{
	Student *s = new Student();
	s->Display();
}